src/main/scala/edu/cmu/cs/obsidian/codegen/yulString.scala

```diff
@@ -0,0 +1,45 @@
+package edu.cmu.cs.obsidian.codegen
+
+import java.io.{FileReader, StringWriter}
+import com.github.mustachejava.Mustache
+import com.github.mustachejava.MustacheFactory
+import com.github.mustachejava.DefaultMustacheFactory
+import edu.cmu.cs.obsidian.codegen.{FuncScope, FunctionCall, FunctionDefinition, Literal, ObjScope, YulObject}
+
+object yulString {
```

Comment:
Should be upper case (https://docs.scala-lang.org/style/naming-conventions.html. By why isn't this a method on YulObject?

Reply:
Originally I was trying to separate it so it becomes more modular. But I agree, making it a method of YulObject is better.